aev/src/main/java/com/fingerprintjs/android/aev/config/ConfigProviderImpl.kt

```diff
@@ -0,0 +1,11 @@
+package com.fingerprintjs.android.aev.config
+
+internal class ConfigProviderImpl(
+
+): ConfigProvider {
+
+    override fun getConfig(): Config {
+        Thread.sleep(50) // emulating file read
```

Comment:
Forgot to remove, uh? 😄

Reply:
This is a dummy implementation anyway :) But okay, will remove it :)